bcipy/acquisition/protocols/lsl/lsl_client.py

```diff
@@ -98,6 +100,7 @@ def first_sample_time(self) -> float:
         """Timestamp returned by the first sample. If the data is being
         recorded this value reflects the timestamp of the first recorded sample"""
         if self.recorder:
+            # TODO: wait until there is a value
```

Comment:
What were you thinking of here?

Reply:
This is a reminder to myself when I look at https://www.pivotaltracker.com/story/show/185952688. If I set a longer wait interval in the recorder then this first timestamp may not be available until after the first chunk has been pulled. I can remove the TODO and just add a note to the ticket.